test/units/testsuite-64.sh

```diff
@@ -588,14 +588,17 @@ testcase_mdadm_basic() {
     udevadm wait --settle --timeout=30 "$raid_dev"
     mkfs.ext4 -L "$part_name" "$raid_dev"
     udevadm wait --settle --timeout=30 "${expected_symlinks[@]}"
-    # Disassemble the array
-    mdadm -v --stop "$raid_dev"
-    udevadm settle
-    helper_check_device_symlinks
-    # Reassemble it and check if all required symlinks exist
-    mdadm --assemble "$raid_dev" --name "$raid_name" -v
-    udevadm wait --settle --timeout=30 "${expected_symlinks[@]}"
-    helper_check_device_symlinks
+    for i in {0..9}; do
+        echo "Disassemble - reassemble loop, iteration #$i"
+        # Disassemble the array
+        mdadm -v --stop "$raid_dev"
+        udevadm settle
+        helper_check_device_symlinks
+        # Reassemble it and check if all required symlinks exist
+        mdadm --assemble "$raid_dev" --name "$raid_name" -v
+        udevadm wait --settle --timeout=30 "${expected_symlinks[@]}"
+        helper_check_device_symlinks
+    done
     # Cleanup
     mdadm -v --stop "$raid_dev"
     udevadm settle
```

Comment:
Let's also check if all symlinks are removed:
`udevadm wait --settle --timeout=30 --removed $expected_symlinks`

Reply:
Hah, that's a nice feature!

Reply:
Not updated...

Reply:
I'll go through the file and add it to other test cases as well.

Reply:
Ah, nevermind, looks you've already taken care of that :-)